extra/swagger.js

```diff
@@ -0,0 +1,25 @@
+/* npm run swagger-autogen */
+
+const swaggerAutogen = require("swagger-autogen")();
+
+const doc = {
+    info: {
+        title: "Uptime Kuma API",
+        description: "Autogen by swagger-autogen",
+    },
+    host: "localhost:3001",
+    schemes: [ "http" ],
+};
+
+// The working directory is root
+const outputFile = "./swagger.json";
```

Comment:
```suggestion
const outputFile = "./openapi.json";
```
If 3.0.0 is used, it is suggested by the spec to use `openapi.json` as the file name